.github/workflows/dart.yml

```diff
@@ -2,16 +2,24 @@ name: Dart CI
 
 on: [push]
 
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    container:
-      image: cirrusci/flutter:v1.12.13-hotfix.8
+env:
+  DEVELOPER_DIR: /Applications/Xcode_11.3.1.app/Contents/Developer
```

Comment:
Do we have any control over the XCode version?

Reply:
Yes:

- https://github.com/actions/virtual-environments/blob/master/images/macos/macos-10.15-Readme.md
- https://www.jessesquires.com/blog/2020/01/06/selecting-an-xcode-version-on-github-ci/